packages/ocr-service/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@wdio/ocr-service",
+  "author": "Wim Selles - wswebcreation",
+  "description": "A WebdriverIO service that is using Tesseract OCR for Desktop/Mobile Web and Mobile Native App tests.",
+  "version": "0.1.0",
+  "license": "MIT",
+  "homepage": "https://webdriver.io/docs/visual-testing",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webdriverio/visual-testing.git"
+  },
+  "bin": {
+    "ocr-service": "./dist/cli.js"
```

Comment:
usually for CLIs I create a `bin/index.js` file that imports `../dist/cli.js` because afaik these files need to be excutedable (e.g. `chmod +x ./bin/index.js`) which we can't do for compiled files. Can you verify this works, maybe this is not needed anymore?